cookbooks/app-react-people/src/pages/SelectorPage.tsx

```diff
@@ -1,8 +1,129 @@
+import { useCallback, useReducer } from 'react';
+import {
+    PersonInfo,
+    PersonListItem,
+    PersonSelect,
+    PersonSelectEvent,
+} from '@equinor/fusion-react-person';
+import { styled } from 'styled-components';
+
+import { Button, Icon } from '@equinor/eds-core-react';
+import { delete_to_trash } from '@equinor/eds-icons';
+Icon.add({ delete_to_trash });
+
+import { createReducer, createAction } from '@equinor/fusion-observable';
+
+const Story = styled.article`
+    margin: 3em 0;
+`;
+
+/** Actions for reducer cases */
+const actions = {
+    add: createAction('add_selected', (person: PersonInfo) => ({ payload: person })),
+    remove: createAction('remove_person', (id: string) => ({ payload: id })),
+    clear: createAction('clear_selected'),
+};
+
+/** initial users to display in reducer */
+const initial: Record<string, PersonInfo> = {
+    'f59e967d-8422-41ae-8980-a47f3ac0b70c': {
+        mail: 'handah@equinor.com',
+        name: 'Hans Våga Dahle',
+        jobTitle: 'Prin Analyst Digital SW Arch',
+        department: 'FOS FOIT PDP',
+        mobilePhone: '+47 93042741',
+        officeLocation: 'ST-FV E3',
+        upn: 'HANDAH@equinor.com',
+        accountType: 'Employee',
+        azureId: 'f59e967d-8422-41ae-8980-a47f3ac0b70c',
+    },
+    'cbc6480d-12c1-467e-b0b8-cfbb22612daa': {
+        mail: 'OROC@equinor.com',
+        name: 'Odin Thomas Rochmann (Bouvet ASA)',
+        jobTitle: 'X-Bouvet ASA (PX)',
+        department: 'FOIT CON PDP',
+        mobilePhone: '+47 97574503',
+        upn: 'OROC@equinor.com',
+        accountType: 'Consultant',
+        managerAzureUniqueId: '0c30c249-2f02-4fbc-abfa-77cbddeb4230',
+        azureId: 'cbc6480d-12c1-467e-b0b8-cfbb22612daa',
+    },
```

Comment:
Should we anonymize this data - create fictive persone? I have heard `Tux Penguin` from Antarctica and `Ada Lovelace` likes to be used in examples.